tests/write_font_test.py

```diff
@@ -42,6 +42,40 @@ def test_keep_glyph_names(svgs, color_format, keep_glyph_names):
         assert ufo.glyphOrder != ttfont.getGlyphOrder()
 
 
+@pytest.mark.parametrize("svgs", [("rect.svg", "one-o-clock.svg")])
+@pytest.mark.parametrize(
+    "color_format",
+    [
+        "cff_colr_0",
+        "cff_colr_1",
+        "glyf_colr_0",
+        "glyf_colr_1",
+        "picosvg",
+    ],
+)
+@pytest.mark.parametrize("version_major", [12, 9])
+@pytest.mark.parametrize("version_minor", [0, "008"])
+# Test version 16.28
+def test_version_minor(svgs, color_format, version_major, version_minor):
+    config, glyph_inputs = test_helper.color_font_config(
+        {
+            "color_format": color_format,
+            "version_major": 16,
+            "version_minor": 28,
+        },
+        svgs,
+    )
+    ufo, ttfont = write_font._generate_color_font(config, glyph_inputs)
+    ttfont = test_helper.reload_font(ttfont)
+
+    if version_major:
+        assert ufo.info.versionMajor == 16
+    elif version_minor:
+        assert ufo.info.versionMinor == 28
+    # else:
+    # TODO: test default version 1.000
```

Comment:
you may as well add a separate test for that

Reply:
Add `test_default_version`